ov/ov_test.go

```diff
@@ -33,22 +33,22 @@ type OVTest struct {
 }
 
 // get Environment
-func (ot *OVTest) GetEnvironment() {
+func (ot *OVTest) GetEnvironment(env string) {
 	if os.Getenv("ONEVIEW_TEST_ENV") != "" {
 		ot.Env = os.Getenv("ONEVIEW_TEST_ENV")
 		return
 	}
-	ot.Env = "dev"
+	ot.Env = env
```

Comment:
OK, that makes sense... so instead of environments, we are really saying we can purpose this as a way to select test specific to a particular unit case where we can start separating.... I think your way might actually be cleaner, it's to bad go doesn't support defaults.  Then we wouldn't need to pass arguments like "dev" every time.   OK, I'll accept that as a good way to go +1.

I'm going to merge my other PR for the make process, this should allow you to rebase and try this build again.